packages/mask/src/manifest-v3.json

```diff
@@ -13,7 +13,7 @@
     "description": "The portal to the new & open Internet. Send encrypted message and decentralized Apps right on top of social networks.",
     "web_accessible_resources": [
         {
-            "resources": ["js/*", "*.svg", "*.png", "*.css"],
+            "resources": ["js/*", "*.svg", "*.png", "*.css", "https://gateway.ipfscdn.io/ipfs/*"],
```

Comment:
I'm afraid this change causes prevent users automatic upgrading the extension.

Reply:
We already did once when we refactored our Icons